### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by woven-baker**

### CODE REVIEW

File: 02_more_strings_and_control_flow/02_challenge/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Challenge 2
 
-In the `factorial.py` file, write a program that takes a non-negative integer from the user and compute the factorial of this given number. The factorial of a number `n` — denoted by `n!` — is the product of all positive integers less than or equal to n. The factorial function is defined as `n! = n x (n - 1) x (n - 2) x ... x 3 x 2 x 1`
+In the `factorial.py` file, write a program that takes a non-negative integer from the user and computes the factorial of this given number. The factorial of a number `n`, denoted by `n!`, is the product of all positive integers less than or equal to n. The factorial function is defined as `n! = n x (n - 1) x (n - 2) x ... x 3 x 2 x 1`. Please note that the dactorial of `0` is actually `1`. Make sure your program is able to handle this case.
```
Comment (dancer-yuma) [inline 02_more_strings_and_control_flow/02_challenge/README.md:3]: Typo

```suggestion
In the `factorial.py` file, write a program that takes a non-negative integer from the user and computes the factorial of this given number. The factorial of a number `n`, denoted by `n!`, is the product of all positive integers less than or equal to n. The factorial function is defined as `n! = n x (n - 1) x (n - 2) x ... x 3 x 2 x 1`. Please note that the factorial of `0` is actually `1`. Make sure your program is able to handle this case.
```

**Review by dancer-yuma — CHANGES_REQUESTED** [review]
@woven-baker 
Most of the things look good to me, there is just one typo.

**Comment by woven-baker** [comment]
@dancer-yuma I fixed the conflicts. Madi changed a lot of stuff in main to it got a bit complicated. Should be all fixed now.

**Review by dancer-yuma — APPROVED** [review]
@woven-baker 
Looks good to me!

**Pull request merged**